package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "4.0.0-activity-tracker-unauthorised-beta",
+  "version": "4.1.6",
```

Comment:
There should be an entry for this version in `RELEASE-NOTES.md`.

Reply:
will be made yes-once this is commited into master